device/cuda/src/cca/component_connection.cu

```diff
@@ -986,8 +983,11 @@ std::tuple<vecmem::unique_alloc_ptr<unsigned[]>, std::size_t> partition_gpu(
      * it is sometimes desirable to process more than one cell per thread. This
      * slots variable determines the number of cells that is examined per
      * block.
+     * Slots should always be larger or equal to max_cells_per_partition.
      */
-    const unsigned slots = 512;
+    const float mult = 1.9;
```

Comment:
May not be so important but why 1.9 here?

Reply:
It's an interesting behaviour of this algorithm.
If you chose an integer number, because most partitions end up being just a bit smaller than the maximum allowed value, this will result in every thread block having to create a very small partition with it's last few elements.
Choosing something like 1.9, 2.9, etc. results in the final partition still being close to the maximum.

Reply:
I'm currently playing around a bit more with this and I hope to come back to this by seing how the `mult` value affects performance between this algorithm and the subsequent clustering.

Reply:
OK. but we have to be careful if the optimal `mult` is specific to the generic (trackML) detector

Reply:
In general I expect this behaviour to be true in any scenario, altough it can be finetuned depending on the detector and event characteristics.